example.py

```diff
@@ -12,13 +12,15 @@
 
 async def cli():
     if len(sys.argv) < 2:
-        print("Error! Run with <ip> or <ip> <user> <pass>")
+        print("Error! Run with <ip> <init> or <ip> <init> <user> <pass>")
         return
 
     ip = sys.argv[1]
-    if len(sys.argv) > 3:
-        username = sys.argv[2]
-        password = sys.argv[3]
+    init = bool(sys.argv[2] in ["True", "true", "1"])
```

Comment:
We should migrate to argparse https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/zwave_js_server/__main__.py#L15-L30

Reply:
Migrated, thanks